app/controllers/documents_controller.rb

```diff
@@ -145,12 +145,12 @@ def enrollment_verification
   def fed_hub_request
     authorize HbxProfile, :can_call_hub?
 
-    request_hash = {person_id: @person.id, verification_type: @verification_type.type_name}
+    request_hash = { person_id: @person.id, verification_type: @verification_type.type_name }
     result = ::Operations::CallFedHub.new.call(request_hash)
     key, message = result.failure? ? result.failure : result.success
-
     if result.failure?
       @verification_type.fail_type
+      ::Operations::Eligibilities::BuildFamilyDetermination.new.call(family: @person.primary_family, effective_date: TimeKeeper.date_of_record)
```

Comment:
This request can fail for multiple reason. But as per current ticket requirement if the failure is for no VLP documents error then reset the due date.